src/components/input/Input.tsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { Eye, EyeSlash } from '@phosphor-icons/react';
+
+interface InputComponentProps {
+  type?: 'text' | 'password';
+  size?: 'lg' | 'md';
+  disabled?: boolean;
+  accentColor?: 'red' | 'orange' | 'green';
+  placeholder?: string;
+}
+
+const Input = ({
```

Comment:
How can we obtain the written text here?

Reply:
value and onChange added